comp/core/tagger/impl-remote/remote.go

```diff
@@ -356,8 +356,8 @@ func (t *remoteTagger) DogstatsdCardinality() types.TagCardinality {
 // Subscribe returns a channel that receives a slice of events whenever an entity is
 // added, modified or deleted. It can send an initial burst of events only to the new
 // subscriber, without notifying all of the others.
-func (t *remoteTagger) Subscribe(subscriptionID string, filter *types.Filter) (types.Subscription, error) {
-	return t.store.subscribe(subscriptionID, filter)
+func (t *remoteTagger) Subscribe(string, *types.Filter) (types.Subscription, error) {
+	return nil, errors.New("subscription to the remote tagger is not currently supported")
```

Comment:
Thanks for updating this, can you please also update the method comment to mention why Subscription is not supported here? Keeping the method comment as is might fool developers to think the feature is available and functional.